node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java

```diff
@@ -182,7 +182,7 @@ public InfrastructureVersions infrastructureVersions() {
         DefaultTimes(Zone zone) {
             failGrace = Duration.ofMinutes(60);
             periodicRedeployInterval = Duration.ofMinutes(30);
-            redeployMaintainerInterval = Duration.ofMinutes(1);
+            redeployMaintainerInterval = zone.environment().isTest() ? Duration.ofDays(Long.MAX_VALUE) : Duration.ofMinutes(1);
```

Comment:
I wonder if that duration is too long. It used to be capped at Long.MAX_VALUE millis, I think.

Reply:
Yes.

Reply:
Fixed by setting duration to 1 day